Js/script.js

```diff
@@ -0,0 +1,138 @@
+// ============ Mobile Pop up Menu ==========================
+function toggleMenu() {
+  const menuIcon = document.querySelector('.nav-icon');
+  const menu = document.querySelector('.nav-links-container');
+  const screenWidth = window.innerWidth || document.documentElement.clientWidth;
+
+  menuIcon.classList.toggle('open');
+
+  if (menuIcon.classList.contains('open') && screenWidth <= 768) {
+    menu.classList.add('open');
+    menuIcon.innerHTML = '<i class="fa fa-times fa-lg"></i>';
+  } else {
+    menuIcon.innerHTML = '<i class="fa fa-bars fa-lg"></i>';
+    menu.classList.remove('open');
+  }
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const menuLinks = document.querySelectorAll('.nav-links-container a');
+  for (let i = 0; i < menuLinks.length; i += 1) {
+    menuLinks[i].addEventListener('click', () => {
+      toggleMenu();
+    });
+  }
+});
+
+// ====== Storing dynamic content of Instructors section in variables ===========
+const featuredInstructors = [
+  {
+    name: 'Gordon Hinckley',
+    position: 'Photography Instructor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card1',
+  },
+  {
+    name: 'Dickson Joseph',
+    position: 'Photography Instructor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card2',
+  },
+  {
+    name: 'Grace Ching',
+    position: 'Photography Instructor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card3',
+  },
+  {
+    name: 'Victor Omondi',
+    position: 'Studio Instructor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card4',
+  },
+  {
+    name: 'Margaret Wanjiru',
+    position: 'Student Advisor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card5',
+  },
+  {
+    name: 'Sandra Mwende',
+    position: 'Student Advisor',
+    image: './assets/person2.jpg',
+    description: 'Established instructor with over 50 collections sold worldwide.',
+    className: 'card6',
+  },
+
+];
```

Comment:
- [ Required ] Good job generating the _featured speakers_ list dynamically, however, I noticed that all the speakers use the same image which is not very professional. I think it would be better if each speaker had a unique image, I believe this will also give your project a more professional look. Please rectify this.

## Your design
![Screenshot from 2023-06-01 04-18-31](https://github.com/Izebiafe/videography-capstone/assets/16392046/15a56818-6678-4c9e-8304-e27c5941e36c)

## Behance design
![Screenshot from 2023-06-01 04-20-35](https://github.com/Izebiafe/videography-capstone/assets/16392046/2c8a5d2c-3bb4-40a3-a41c-1df0cfc422c1)